File: Java/0001-1000/001-100/21-30/26.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length == 0)
            return 0;
        int cur = nums[0], now = 1;
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] != cur) {
                nums[now] = nums[i];
                cur = nums[i];
                now++;
            }
        }
        return now;
    }
}

File: Python/0001-1000/001-100/21-30/26.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = len(nums)
        if n < 2:
            return n
        i = 1
        now = 0
        for i in range(1, n):
            if nums[i] != nums[now]:
                now += 1
                nums[now] = nums[i]
        return now + 1
